lib/ansible/modules/system/known_hosts.py

```diff
@@ -240,7 +243,7 @@ def search_for_host_key(module,host,key,hash_host,path,sshkeygen):
 
     sshkeygen_command.insert(1,'-H')
     rc,stdout,stderr=module.run_command(sshkeygen_command,check_rc=False)
-    if rc!=0: #something went wrong
+    if rc not in (0, 1) or stderr != '': #something went wrong
```

Comment:
I feed kind of bad that I didn't report the bug upstream, but "OpenSSH" makes me think "maintained by OpenBSD guys", which leads to "probably in some CVS repository from 1998 with a mailing list instead of a bug tracker", and I don't have the energy.

(Also I'm probably wrong and they use Git, or at least Subversion now.  Still.)